### PR TITLE
[WIP] Mktridactylrc

### PR DESCRIPTION
**Pull request by antonva**
@bovine3dom here you go.  As per #695 

It currently does not write anything to file but it parses the config pretty decently.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -466,6 +469,75 @@ export async function update() {
...
+        } else {
+            for (let e in USERCONFIG[i]) {
+                parsedConf.push(`set ${i}.${e} ${USERCONFIG[i][e]}`)
+            }
+        }
+    }
+
+    let configFile = ""
+    if (parsedConf.length > 0)
+        configFile += "//General Settings\n" + parsedConf.join("\n") + "\n\n"
```
Comment (bovine3dom) [inline src/config.ts]: Comments start with " in Vim : )

Comment (antonva) [reply src/config.ts]: Hahah, yes they do :)

**Comment by antonva** [comment]
Yes I have some more commits and plans for this. Just got sidetracked with trying to get some sort of e2e testing operational.

**Comment by antonva** [comment]
Closing this in favour of a new PR at a later date.

**Pull request closed without merging**